site/en/docs/extensions/reference/offscreen/index.md

```diff
@@ -48,19 +48,26 @@ chrome.offscreen.createDocument({
 chrome.offscreen.closeDocument()
 ```
 
-The following example shows how to ensure that the offscreen document has already been created. The `setupOffscreenDocument()` function calls [`clients.matchAll()`](https://developer.mozilla.org/docs/Web/API/Clients/matchAll) to find the existing offscreen document or creates it if it doesn't already exist. Note that an extension can only have one offscreen document.
+The following example shows how to ensure that the offscreen document has already been created. The `setupOffscreenDocument()` function calls [`runtime.getContexts()`][runtime-get-contexts] to find the existing offscreen document or creates it if it doesn't already exist. Note that an extension can only have one offscreen document.
+
+{% Aside 'gotchas' %}
+[`runtime.getContexts()`][runtime-get-contexts] was added in Chrome 116. In earlier versions of
+Chrome, you can check for the existence of the offscreen document using [`clients.matchAll()`](https://developer.mozilla.org/docs/Web/API/Clients/matchAll).
```

Comment:
I'd be helpful to include the snippet here as it's not straightforward how to use it for an offscreen doc.

Reply:
I'd simply add the following snippet to the aside:

```
const matchedClients = await clients.matchAll();
   for (const client of matchedClients) {
     if (client.url === offscreenUrl) {
       return;
     }
   }
```

Reply:
Done!